common/src/java/com/zimbra/common/util/ZimbraCookie.java

```diff
@@ -70,7 +70,14 @@ private static void addCookie(HttpServletResponse response, String name, String
         }
         ZimbraCookie.setAuthTokenCookieDomainPath(cookie, ZimbraCookie.PATH_ROOT);
 
-        cookie.setSecure(secure);
+        String cookieVal = LC.zimbra_same_site_cookie.value();
```

Comment:
hope this is used in JWT authentication also, can we check it?

Reply:
can we also add unittests for this change?

Reply:
@silentsakky Above method is also called for JWT token also can see here https://github.com/Zimbra/zm-mailbox/blob/develop/store/src/java/com/zimbra/cs/account/ZimbraJWToken.java#L301 but need to verify by doing testing.

Reply:
Thanks @silentsakky  for you help verifying feature for JWT token, as you shared and I also verified that this is working for JWT token too. 
![JWT Strict](https://user-images.githubusercontent.com/38309486/152172385-a5a50a60-b04d-4073-929c-3213a8e1b27f.png)